File: app/src/main/java/com/tson/utils/entity/BaseEntity.java
package com.tson.utils.entity;

/**
 * Date 2019/5/27 11:26 AM
 *
 * @author tangxin
 */
public class BaseEntity<T> {

    /**
     * The Result.
     */
    public boolean result;
    /**
     * The Reason.
     */
    public String reason;
    /**
     * The Package set.
     */
    public T package_set;

}

File: app/src/main/java/com/tson/utils/entity/BaseEntity.kt
package com.tson.utils.entity

/**
 * Date 2019/5/27 11:26 AM
 *
 * @author tangxin
 */
class BaseEntity<T> {

    /**
     * The Result.
     */
    var result: Boolean = false
    /**
     * The Reason.
     */
    var reason: String? = null
    /**
     * The Package set.
     */
    var package_set: T? = null

}
